include/ur_client_library/rtde/rtde_client.h

```diff
@@ -208,7 +208,7 @@ class RTDEClient
   constexpr static const double CB3_MAX_FREQUENCY = 125.0;
   constexpr static const double URE_MAX_FREQUENCY = 500.0;
 
-  std::vector<std::string> readRecipe(const std::string& recipe_file);
+  std::vector<std::string> readRecipe(const std::string& recipe_file, bool output_recipe = false);
```

Comment:
Suggest we a using a an enum so it is more readable to that the boolean is doing.
And comments won't hurt either